api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -201,11 +201,16 @@ default AppendFiles newFastAppend() {
 
   /**
    * Create a new {@link Rollback rollback API} to roll back to a previous snapshot and commit.
-   *
    * @return a new {@link Rollback}
    */
   Rollback rollback();
```

Comment:
Can we deprecate this method as well? (And also add the newline back in the javadoc to avoid unnecessary conflicts)